keyboards/handwired/co60/rev1/rev1.h

```diff
@@ -0,0 +1,127 @@
+/* Copyright 2018 John M Daly
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// The following is a layout that uses all available switch positions.
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define LAYOUT_all( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, \
+    K100,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, \
+    K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, \
+    K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311,       K313, K314, \
+    K400, K401, K402,       K404,       K406,       K408,       K410, K411, K412, K413, K414 \
```

Comment:
**The issues in this file are also present in `rev6.h` and `rev7.h`.**

You have 68 keys defined here, but looking over your other macros and keymaps, I don't think `K412` should be here; none of your other macros use it, and your `info.json` only shows 4 keys to the right of the (split) Spacebar.

```suggestion
    K400, K401, K402,       K404,       K406,       K408,       K410, K411,       K413, K414 \
```